exercises/practice/game-of-life/.meta/example.ua

```diff
@@ -0,0 +1 @@
+Tick ← ↥⊙↧∩=3,2-,/+≡⬚0↻☇1-1⇡3_3¤.
```

Comment:
That's mighty compact. I looked at haskell, prolog, julia and f# to see their solutions, but nothing comes even close.